packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -374,6 +374,17 @@ export function createLodestarMetrics(
       help: "Total count state.validators nodesPopulated is false on stfn for post state",
     }),
 
+    epochCache: {
+      finalizedPubkeyDuplicateInsert: register.gauge({
+        name: "lodestar_epoch_cache_finalized_pubkey_duplicate_insert",
+        help: "Total count of duplicate insert of finalized pubkeys",
+      }),
+      newUnFinalizedPubkey: register.gauge({
+        name: "lodestar_epoch_cache_new_unfinalized_pubkey",
```

Comment:
Which should try to follow [best practices](https://prometheus.io/docs/practices/naming/) for new metrics, this will also improve consistency

```suggestion
        name: "lodestar_epoch_cache_new_unfinalized_pubkey_total",
```